win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -1067,14 +895,14 @@ TEST_FUNCTION(chaos_knight_test)
     }
 
     // assert that all scheduled items were executed
-    ASSERT_ARE_EQUAL(int64_t, (int64_t)InterlockedAdd64(&chaos_test_data.expected_call_count, 0), (int64_t)InterlockedAdd64(&chaos_test_data.executed_work_functions, 0));
+
+    InterlockedHL_WaitForValue64(&chaos_test_data.executed_work_functions, chaos_test_data.expected_call_count, UINT32_MAX);
```

Comment:
>InterlockedHL_WaitForValue64 [](http://example.com/codeflow?start=4&length=28)

We should ASSERT on the result of this, right?

Reply:
Fixed.